bitnami/redis/Chart.yaml

```
@@ -25,4 +25,4 @@ name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
   - http://redis.io/
-version: 15.3.2
+version: 15.3.3
```

As there are only changes in the README it is not needed to bump the chart version nor perform a new release
```suggestion
version: 15.3.2
```